src/interface.jl

```diff
@@ -191,6 +191,10 @@ julia> collect(edges(g))
  Edge 1 => 2
  Edge 2 => 3
 ```
+
+### See also
```

Comment:
We don't really define a `### See also` section anywhere else. Let me think about the best way of doing this.